leaflet-treering.js

```diff
@@ -553,13 +554,13 @@ function Autoscroll (viewer) {
 function MarkerIcon(color, LtBasePath) {
 
   var colors = {
-    'light_blue': { 'path': '/assets/leaflet-treering/images/light_blue_tick_icon.png',
+    'light_blue': { 'path': Lt.meta.imagePath + '/images/light_blue_tick_icon.png',
```

Comment:
I think we should also trip the leading "/" from images. 
With web urls, if you don't have that "/" in front, it tells the browser to look for the files relative to the current file.  For example, if the index file was at:
http://www.myserver.com/leaflet-treering/index.html
Right now, the browser would look for the images at:
http://www.myserver.com/images/light_blue_tick_icon.png
If you remove that leading /, the browser will instead look at:
http://www.myserver.com/leaflet-treering/images/light_blue_tick_icon.png
which is more likely to be correct